### ISSUE TITLE
Homebridge crashes on change

### ISSUE THREAD

**Issue by paqpaqpaq** [F4stFr3ddy/homebridge-jablotron-alarm#10]
```
[2018-12-2 10:19:04] [Alarm System] Is alarm active? false
[2018-12-2 10:19:04] [Alarm System] Is alarm active? false
[2018-12-2 10:19:12] [Alarm System] Switching to state: 1
[2018-12-2 10:19:12] [Alarm System] Using cached sessionId.
[2018-12-2 10:19:15] [Alarm System] { status: false,
  error_status: 'bypass_dialog_timed',
  checksum: '9f785d19e3d329edfb6ebff755a44f79',
  server_id: '4829128-1543742355',
  client_id: null }

/usr/lib/node_modules/homebridge-jablotron/Jablotron/jablotronClient.js:53
                callback(JSON.parse(data));
                ^
Error: bypass_dialog_timed
    at /usr/lib/node_modules/homebridge-jablotron/Jablotron/jablotronClient.js:24:23
    at IncomingMessage.resp.on (/usr/lib/node_modules/homebridge-jablotron/Jablotron/jablotronClient.js:53:17)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)

```

As soon as I engage the alarm, homebridge crashes.

Also, with setup:
this command 
`curl -H 'x-vendor-id: MyJABLOTRON' --data "login=YOUR_USERNAME&password=YOUR_PASSWORD" --compressed 'https://api.jablonet.net/api/1.6/login.json'`
already will give the needed session_id (?)
It has the same id as when using 

> This will give back some info, locate the session_id, it looks like this:

“session_id”:"lol808b3aahej01s7abc1a12f38"
Copy the alphanumeric string, and paste in the below command where it says SESSION_ID, after doing that execute the command

curl -H 'x-vendor-id: MyJABLOTRON'  -H 'Cookie: PHPSESSID=SESSION_ID' --data-binary '{"list_type":"extended"}' --compressed 'https://api.jablonet.net/api/1.6/getServiceList.json'
This will give a reply, locate the following:

{“services":[{"id":1234567,"name":"Your alarm”,"service_type":"JA100"

**Comment by F4stFr3ddy** [comment]
Sorry, with single alarm users I mean; users with only 1 alarm. Not alarms with 1 user. Did you make sure the credentials you are using have enough permissions?

Can you update the package? `npm update -g homebridge-jablotron`

If that doesn't solve then I'll debug a bit more.

**Comment by paqpaqpaq** [comment]
I don't get any feedback on the npm update (no error either).

on `sudo npm install -g homebridge-jablotron` it updates 1 package
after a reboot same behaviour.

Also files like config_helper.py aren't to be found.


we have 2 users on the alarm, each serving it's own section, but both users can arm/disarm both sections, or the overall alarm (section 3)

**Comment by F4stFr3ddy** [comment]
My bad, I forgot to publish to NPM, sorry I'm new to NPM / NodeJS. Please try the update again.

Regarding your users, that should be correct.

**Comment by paqpaqpaq** [comment]
V1.0.1 is up now and updated.
Homebridge doesn't crash, but the plugin does.

the config helper script sends:
`Traceback (most recent call last):
  File "config_helper.py", line 24, in <module>
    r = request(url='https://api.jablonet.net/api/1.6/login.json', headers=headers1, params=params1)
  File "config_helper.py", line 10, in request
    data = json.loads(response)
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
`

**Comment by paqpaqpaq** [comment]
I logged on with the same account as in the app yes
Also I'm positive the service id is correct. (as are the other credentials)
when we make a call to jablonet.net/app/ja100 we can operate it in every way.

**Comment by F4stFr3ddy** [comment]
With the new version 1.0.2 can you try the helper again?

Also, could you try different credentials in the config? I have no clue what the problem could be as we have no documentation from Jablotron. It works for others with this version so I'm guessing its user credentials or version of Node / Raspbian etc.

**Comment by paqpaqpaq** [comment]
py script now functions.
With the correct service ID as outcome.

Still the plugin refuses . It only seems to get the current status but doesn't change it

**Comment by paqpaqpaq** [comment]
When I manually arm the alarm, the plugin shows the correct status.
When I disarm it in homekit, it says it did so, but it really doesn't.
When I manually disarm the alarm, the plugin again shows the correct status.

**Comment by F4stFr3ddy** [comment]
Did you try the latest version? I'll close the issue, please re-open if the issue is still present and you want me to further debug.

**Comment by paqpaqpaq** [comment]
Awesome!  works great on 1.0.3

**Comment by paqpaqpaq** [comment]
> Did you try the latest version? I'll close the issue, please re-open if the issue is still present and you want me to further debug.

Sorry... it will only work on section_1
On section_2 nothing happens (although it says so) and cannot disengage,

**Comment by F4stFr3ddy** [comment]
Just a heads-up your issue has been solved in the new version (2.0.0) its currently in a dev branch, expect to publish it on NPM within 3 days. The new version supports multiple zones as separate alarms in the Home app.
